package.json

```diff
@@ -2,11 +2,16 @@
   "name": "redux-promise-actions",
   "version": "0.5.1",
   "description": "Flux Standard Action async utilities for Redux.",
-  "main": "index.js",
+  "main": "dist",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "build": "babel src --presets env --out-dir dist",
     "lint": "eslint src/**/*.js",
+    "prepublish": "npm run build",
     "report-coverage": "nyc report --reporter=lcov > coverage.lcov && codecov",
-    "test": "npm run lint && npm run test:only",
+    "test": "npm run build && npm run lint && npm run test:only",
```

Comment:
I don't see a lot of value in adding the build step to the test step. The tests are still written against the un-built version of the code, and if the build fails, it'll fail before publish and stop broken modules from being published anyway. I'd say remove this, but I'm open to hearing reasons for keeping it.